casdk-docs/blog/2024-05-16-dotnet8/index.md

```diff
@@ -0,0 +1,156 @@
+
+The Carbon Intensity can be intuitively visualized with size and colors on a geomap, beyond raising awareness, this can guide decisions on location or time shift.
+
+![fig3](./fig3.png)
+
+## Green Dashboard for Kubernetes
+
+Carbon Aware SDK helps increase awareness around Carbon emission, and it is now possible to monitor carbon emission per application within Kubernetes.
+
+In practice, each container energy consumption is evaluated through [Kepler](https://www.cncf.io/projects/kepler/) (sandbox project in Cloud Native Cloud Foundation, [CNCF](https://www.cncf.io/)), and thanks to the Carbon Aware SDK, the carbon emission can be evaluated.
+
+All those emission data from power grid can be accessed through Prometheus exporter with Carbon Aware SDK (starting v1.4.0), and a visualized with Grafana dashboard.
+
+The power consumption, energy consumption, carbon emission, and SCI can be seen at a glance!
+
+One of the upside of micro-services architecture, as Kubernetes facilitates it, is to be able to work on different piece of an application in a relatively independant fashion (maintaining, scaling, optimizing…).
+
+The Green Dashboard allows a team to check an application global energy consumption and carbon emission (dashboard left side), while drilling down at sustainability-related metrics for each micro-service (dashboard right side).
+
+It shows the SCI, allowing to get a sense of the rate of Carbon Emission down to a particular piece of an architecture (R being the [functional unit](https://learn.greensoftware.foundation/measurement/#software-carbon-intensity-specification) for that service - for example an API call, the data is being shown per R unit or over an hour).
+
+While in monolitic application optimization need customized instrumentation, and often have rippled effect, this green dashboard helps identifying which micro-service refactoring would have the maximum impact on the application carbon footprint, leveraging the team effort more efficiently.
+
+![fig4](./fig4.png)
+
+# Moving Forward
+
+With the Cloud Computing expansion, and Kubernetes flexibility, more and more choices exist for running a workload. While business and economical constraints often lead those decisions, the carbon footprint is becoming an increasingly critical consideration.
+
+This is a difficult endeavour, and the first step is to know where one stands, measure but also later evaluate and confirm what action would lead to improvement. That was one of the intentions behind the Green Dashboard for Kubernetes, and the Carbon Aware SDK 1.4 is key element in this approach.
```

Comment:
```suggestion
This is a difficult endeavor, and the first step is to know where one stands, measure but also later evaluate and confirm what action would lead to improvement. That was one of the intentions behind the Green Dashboard for Kubernetes, and the Carbon Aware SDK 1.4 is a key element in this approach.
```

Reply:
The rest of the article has American spelling so updating this to align.